crates/gen-spidermonkey/src/lib.rs

```diff
@@ -2,10 +2,16 @@
 
 #![deny(missing_docs)]
 
+#[cfg(feature = "structopt")]
+mod opts;
+#[cfg(feature = "structopt")]
```

Comment:
FWIW for the other generators I've been having a conditional derive of `structopt` where the main "builder" has configuration options on it for libraries and such. That may be applicable here too?

Reply:
I just didn't want `SpiderMonkeyWasm` construction to be fallible and do I/O. It feels cleaner to pass in the JS source string as a parameter to the generator, rather than pass the file path and expect it to read it for you.